tools/eslint-rules/no-unescaped-regexp-dot.js

```diff
@@ -4,48 +4,20 @@
  */
 'use strict';
 
-const path = require('path');
-const utilsPath = path.join(__dirname, '..', 'eslint', 'lib', 'ast-utils.js');
-const astUtils = require(utilsPath);
-const getLocationFromRangeIndex = astUtils.getLocationFromRangeIndex;
-const getRangeIndexFromLocation = astUtils.getRangeIndexFromLocation;
-
 //------------------------------------------------------------------------------
 // Rule Definition
 //------------------------------------------------------------------------------
 
 module.exports = function(context) {
-  const sourceCode = context.getSourceCode();
+  const s = context.getSourceCode();
```

Comment:
You don't need to break the object format if you don't want to. You could temporarily store all or part of the `loc` in a separate variable to avoid that.